gossip-base/src/main/java/org/apache/gossip/crdt/LWWSet.java

```diff
@@ -25,7 +25,13 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-public class LWWSet<ElementType> implements CrdtSet<ElementType, Set<ElementType>, LWWSet<ElementType>> {
+/*
+  Last write wins CrdtSet. Each operation has timestamp and the latest wins.
```

Comment:
More documentation is needed here. Linking off to meangirls is ok but we need to describe the implementation here. What it does when to use it, what are issues from clock skew etc.